app/event/send-processing-ack-event.js

```diff
@@ -0,0 +1,15 @@
+const raiseEvent = require('./raise-event')
+const { v4: uuidv4 } = require('uuid')
+
+const sendProcessingAckEvent = async (message, action) => {
+  const event = {
+    id: message?.correlationId ?? uuidv4(),
+    name: `payment-request-${action}-event`,
```

Comment:
Don't think these all need `event` on the end

Reply:
removed from all